XCUITests/ToolbarTest.swift

```diff
@@ -112,18 +112,19 @@ class ToolbarTests: BaseTestCase {
             waitForExistence(app.otherElements["Navigation Toolbar"])
         }
         navigator.openURL(website1["url"]!, waitForLoading: true)
+        // Adding the waiter right after navigating to the webpage in order to make the test more stable
```

Comment:
I would also add here the `waitUntilPageLoad`to start waiting for the element once the page is completely shown.